src/BloomExe/Shell.cs

```diff
@@ -125,13 +125,18 @@ protected override void OnClosing(CancelEventArgs e)
 			base.OnClosing(e);
 		}
 
-		private void SetWindowText()
+		public void SetWindowText(string bookName)
 		{
-			Text = string.Format("{0} - Bloom {1} Built on {2}", _workspaceView.Text, GetShortVersionInfo(), GetBuiltOnDate());
+			string formattedText = string.Format("{0} - Bloom {1} Built on {2}", _workspaceView.Text, GetShortVersionInfo(), GetBuiltOnDate());
+			if (bookName != null)
+			{
+				formattedText = string.Format("{0} - {1}", bookName, formattedText);
+			}
 			if(_collectionSettings.IsSourceCollection)
 			{
-				Text += "SOURCE COLLECTION";
+				formattedText += "SOURCE COLLECTION";
```

Comment:
While we're doing this might be a good time to insert a space between the year and "SOURCE COLLECTION"